### PR TITLE
@xtina-starr => [Onboarding] Add a proper loader for suggested artists

### PR DESCRIPTION
**Pull request by mzikherman**
Still to do: adding in that schema we talked about in https://github.com/artsy/metaphysics/pull/790#issuecomment-339298135

### CODE REVIEW

File: lib/loaders/loaders_with_authentication/index.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ export default (accessToken, userID, requestID) => {
       "is_saved"
     ),
     saleArtworksLoader: gravityLoader(id => `sale/${id}/sale_artworks`),
+    suggestedArtistsLoader: gravityLoader("me/suggested/artists", {}, { headers: true }),
```
Comment (mzikherman) [inline lib/loaders/loaders_with_authentication/index.js]: We want to be able to pluck the total count, so make sure we are able to resolve headers.

File: schema/artist/index.js [hunk 2]
```diff
@@ -439,6 +441,32 @@ export const ArtistType = new GraphQLObjectType({
         type: GraphQLBoolean,
         deprecationReason: "Favor `is_`-prefixed boolean attributes",
       },
+      related: {
```
Comment (mzikherman) [inline schema/artist/index.js]: Adds a new `related` field, which currently just holds `suggested`, which is an artists connection based on suggestions.

**Comment by mzikherman** [comment]
Looks like

screen shot 2017-11-03 at 11 37 20 am


Definitely open to another word besides `related` (since that's already overloaded in our parlance).

**Comment by mzikherman** [comment]
@xtina-starr should be ready to review.

File: schema/artist/index.js [hunk 3]
```diff
@@ -486,7 +514,7 @@ const Artist: GraphQLFieldConfig<ArtistType, *> = {
       type: new GraphQLNonNull(GraphQLString),
     },
   },
-  resolve: (root, { id }, request, resolver) => {
+  resolve: (_root, { id }, _request, resolver) => {
```
Comment (xtina-starr) [inline schema/artist/index.js]: was just going to ask if that s was a type lol.

File: schema/artist/index.js [hunk 4]
```diff
@@ -454,7 +482,7 @@ export const ArtistType = new GraphQLObjectType({
           },
           sort: SaleSorts,
         },
-        resolve: ({ id }, options, request, { rootValue: { relatedSalesLoader } }) =>
+        resolve: ({ id }, options, _request, { rootValue: { relatedSalesLoader } }) =>
```
Comment (xtina-starr) [inline schema/artist/index.js]: It's not apparently obvious where the `_request` is coming from. Is this some sort of convention?

Comment (mzikherman) [reply schema/artist/index.js]: Yes, this signifies that the parameter will be unused (convention is to precede with a `_`).

File: schema/artist/index.js (hunk 3 above)
Comment (mzikherman) [reply schema/artist/index.js]: 🙊

**Pull request merged**